src/screens/Member/styles.ts

```diff
@@ -0,0 +1,111 @@
+import {StyleSheet} from "react-native";
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1, justifyContent: 'center'
+    },
+    textInput: {
+        height: 50,
+        borderColor: '#dbdbdb',
+        borderWidth: 1,
+        margin: 10,
+        marginLeft: 20,
+        marginRight: 20,
+        borderRadius: 3,
+        padding: 10,
+        backgroundColor: "#fafafa"
+    },
+    logo: {
+        fontSize: 50,
+        fontFamily: 'Billabong',
+        marginBottom: 20,
+    },
+    dontHaveAccountView: {
+        alignItems: 'center',
+        height: 50,
+        borderTopWidth: 1,
+        borderColor: '#ddd',
+        justifyContent: 'center'
+    },
+    loginFacebookView: {
+        marginTop: 30,
+        flexDirection: 'row',
+        alignItems: 'center',
+        justifyContent: 'center'
+    },
+    loginFacebookText: {
+        textAlign: 'center',
+        fontWeight: 'bold',
+        fontSize: 15,
+        color: '#385185',
+        paddingLeft: 10
+    },
+    orView: {
+        borderBottomWidth: 1,
+        justifyContent: 'center',
+        alignItems: 'center',
+        marginLeft: 20,
+        marginRight: 20,
```

Comment:
Same here  can be replaced by a single property `marginHorizontal` like `marginHorizonta:20`